modules/workerpool-gke-ha-vpn/main.tf

```
@@ -17,7 +17,7 @@
 # HA VPN
 module "vpn_ha_1" {
   source     = "terraform-google-modules/vpn/google//modules/vpn_ha"
-  version    = "~> 2.3.0"
+  version    = "~> 2.3.0, < 2.3.2"
```

```suggestion
  version    = "~> 2.3.0, != 2.3.2"
```

Better to exclude that exact version, in case they introduce new, _real_ patch releases.